src/pages/simulator/result/index.tsx

```diff
@@ -34,49 +34,47 @@ export const SimulatorResultPage = () => {
   return (
     <div className="p-20">
       {simulationType === 'recurring' && (
-        <Link
-          to="/simulate/recurring"
-          search={{
-            baseToken: ctx.search.baseToken,
-            quoteToken: ctx.search.quoteToken,
-            start: ctx.search.start,
-            end: ctx.search.end,
-            buyMin: ctx.search.buyMin,
-            buyMax: ctx.search.buyMax,
-            buyBudget: ctx.search.buyBudget,
-            sellMin: ctx.search.sellMin,
-            sellMax: ctx.search.sellMax,
-            sellBudget: ctx.search.sellBudget,
-            buyIsRange: ctx.search.buyIsRange,
-            sellIsRange: ctx.search.sellIsRange,
-          }}
-          className="text-24 font-weight-500 mb-16 flex items-center"
-        >
-          <div className="bg-background-800 mr-16 flex size-40 items-center justify-center rounded-full">
-            <IconChevronLeft className="size-16" />
-          </div>
+        <div className="text-24 font-weight-500 mb-16 flex items-center">
+          <Link
+            to="/simulate/recurring"
+            search={{
+              baseToken: ctx.search.baseToken,
+              quoteToken: ctx.search.quoteToken,
+              start: ctx.search.start,
+              end: ctx.search.end,
+              buyMin: ctx.search.buyMin,
+              buyMax: ctx.search.buyMax,
+              buyBudget: ctx.search.buyBudget,
+              sellMin: ctx.search.sellMin,
+              sellMax: ctx.search.sellMax,
+              sellBudget: ctx.search.sellBudget,
+              buyIsRange: ctx.search.buyIsRange,
+              sellIsRange: ctx.search.sellIsRange,
+            }}
+          >
+            <BackButton className="mr-16" />
```

Comment:
We should never put a button inside a link. They are focusable element so it leads to double Tab action. Also interactive element are not permitted as descendant of an anchor : 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary